sympy/matrices/matrices.py

```diff
@@ -4278,59 +4278,239 @@ def entry_U(i, j):
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            L_{n-1, 0} & L_{n-1, 1} & L_{n-1, 2} & \cdots & 1
+            \end{bmatrix}
+
+        .. math::
+            U = \begin{bmatrix}
+            U_{0, 0} & U_{0, 1} & U_{0, 2} & \cdots & U_{0, n-1} \\
+            0 & U_{1, 1} & U_{1, 2} & \cdots & U_{1, n-1} \\
+            0 & 0 & U_{2, 2} & \cdots & U_{2, n-1} \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            0 & 0 & 0 & \cdots & U_{n-1, n-1}
+            \end{bmatrix}
+
+        And for a matrix with more rows than the columns,
+        the decomposition would look like:
+
+        .. math::
+            L = \begin{bmatrix}
+            1 & 0 & 0 & \cdots & 0 & 0 & \cdots & 0 \\
+            L_{1, 0} & 1 & 0 & \cdots & 0 & 0 & \cdots & 0 \\
+            L_{2, 0} & L_{2, 1} & 1 & \cdots & 0 & 0 & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \ddots
+            & \vdots \\
+            L_{n-1, 0} & L_{n-1, 1} & L_{n-1, 2} & \cdots & 1 & 0
+            & \cdots & 0 \\
+            L_{n, 0} & L_{n, 1} & L_{n, 2} & \cdots & L_{n, n-1} & 1
+            & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots
+            & \ddots & \vdots \\
+            L_{m-1, 0} & L_{m-1, 1} & L_{m-1, 2} & \cdots & L_{m-1, n-1}
+            & 0 & \cdots & 1 \\
+            \end{bmatrix}
+
+        .. math::
+            U = \begin{bmatrix}
+            U_{0, 0} & U_{0, 1} & U_{0, 2} & \cdots & U_{0, n-1} \\
+            0 & U_{1, 1} & U_{1, 2} & \cdots & U_{1, n-1} \\
+            0 & 0 & U_{2, 2} & \cdots & U_{2, n-1} \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            0 & 0 & 0 & \cdots & U_{n-1, n-1} \\
+            0 & 0 & 0 & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            0 & 0 & 0 & \cdots & 0
+            \end{bmatrix}
+
+        Finally, for a matrix with more columns than the rows, the
+        decomposition would look like:
+
+        .. math::
+            L = \begin{bmatrix}
+            1 & 0 & 0 & \cdots & 0 \\
+            L_{1, 0} & 1 & 0 & \cdots & 0 \\
+            L_{2, 0} & L_{2, 1} & 1 & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            L_{m-1, 0} & L_{m-1, 1} & L_{m-1, 2} & \cdots & 1
+            \end{bmatrix}
+
+        .. math::
+            U = \begin{bmatrix}
+            U_{0, 0} & U_{0, 1} & U_{0, 2} & \cdots & U_{0, m-1}
+            & \cdots & U_{0, n-1} \\
+            0 & U_{1, 1} & U_{1, 2} & \cdots & U_{1, m-1}
+            & \cdots & U_{1, n-1} \\
+            0 & 0 & U_{2, 2} & \cdots & U_{2, m-1}
+            & \cdots & U_{2, n-1} \\
+            \vdots & \vdots & \vdots & \ddots & \vdots
+            & \cdots & \vdots \\
+            0 & 0 & 0 & \cdots & U_{m-1, m-1}
+            & \cdots & U_{m-1, n-1} \\
+            \end{bmatrix}
+
+        About the compressed LU storage:
```

Comment:
Is it possible to have a sub-header like
```
Compressed LU storage
---------------------
```
rather than writing "About the"?

Reply:
I have tried this before and turned out to be automatically decapitalized like `Compressed Lu Stroage`

I would resolve the doc builder issue first.